lib/connect_proto/src/result.proto

```diff
@@ -86,7 +86,7 @@ message Result {
     string url = 3;
   }
 
-  CodedValue code = 1; // Code for the result component
+  CodedValue code = 1; // Deprecating in favor of coded_value (21)
```

Comment:
this isn't the one that's deprecated, its `value =4` that is getting replaced, isn't it? And maybe it's _not_ actually replaced? Maybe it's only for codable values? E.g. if a result is 5.1, it still goes in `value`, but if it's `{code_set: Snomed, code: XXX}` then it goes in `coded_value`?

Reply:
Yes, `value = 4` is for non-coded result values so that is staying as-is. Currently, `code = 1` is what we're using for results that are formatted like `{code_set: SCT, value: XXX}` so that's what is getting "replaced" by `coded_value`. It's essentially a name change, but perhaps its not necessary?

Reply:
Oh, right... `code` is currently misused. 👍